docs/pages/admin-guides/access-controls/access-lists/nested-access-lists.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Nested Access Lists
+description: Learn how to use nested Access Lists to manage complex permissions and grant inheritance in Teleport.
+---
+
+This guide will help you:
+
+- Understand how nesting and inheritance work in Access Lists
+- Create a nested Access List
+- Verify inherited permissions granted through the nested Access List
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- A user with the `editor` role or equivalent permissions to create and manage Access Lists.
+- Familiarity with basic Access List concepts (see the [Getting Started with Access Lists guide](./guide.mdx)).
+- At least one user with only the `requester` to add to the Access List.
+- At least one application or resource to grant access to.
+
+## What Are Nested Access Lists?
+
+Nested Access Lists allow inclusion of an Access List as a member or owner of another Access List.
+This enables hierarchical permission structures where permissions can be inherited from multiple levels of
+parent Access Lists.
```

Comment:
I think we should make this the first paragraph of the guide to define nested Access Lists for someone who has just visited this page.